core/src/main/java/com/redhat/lightblue/client/request/AbstractLightblueDataExecutionRequest.java

```diff
@@ -0,0 +1,28 @@
+package com.redhat.lightblue.client.request;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.redhat.lightblue.client.Execution;
+
+public abstract class AbstractLightblueDataExecutionRequest extends AbstractLightblueDataRequest {
```

Comment:
I still think an interface might be better. appendExecutionToJson could be a static method on the interface.

If the method "execution" returned AbstractLightblueDataRequest, then each implementation could return it's own class sub-type.

This just means each implementation would need to have it's own Execution instance, but that precedence has already been set with Projection and whatnot. We could go back and change appendRangeToJson to be the same in another PR.